ReactNativeClient/ios/Joplin.xcodeproj/project.pbxproj

```diff
@@ -42,6 +44,7 @@
 		AE6BB3A2FDA34D01864A721A /* libRNVectorIcons.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 381C047F2739439CB3E6452A /* libRNVectorIcons.a */; };
 		AE98927FE9BF483FB24D229F /* libRCTImageResizer.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 508DD20D1EA341CA8F730F22 /* libRCTImageResizer.a */; };
 		AF99EEC6C55042F7BFC87583 /* libRNImagePicker.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 44A39642217548C8ADA91CBA /* libRNImagePicker.a */; };
+		BEECB8BAF419407AB74CE211 /* libRNQuickAction.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 97C7168EEE364A80B33E87BE /* libRNQuickAction.a */; };
```

Comment:
Note to reviewer: I didn't touch this file; these lines were added automatically when I installed `react-native-quick-actions` via npm and linked the project in xcode (as described in the OP of this PR)